examples/swarmion-full-stack/swarmion-starter.code-workspace

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Change this filename to `examples/swarmion-full-stack/swarmion-full-stack.code-workspace`